ui/src/app/store/machine/actions.test.ts

```diff
@@ -616,6 +617,72 @@ describe("machine actions", () => {
     });
   });
 
+  it("can handle creating a physical interface", () => {
+    expect(
+      actions.createPhysical({
+        enabled: true,
+        interface_speed: "10",
```

Comment:
Shouldn't `interface_speed` and `link_speed` be numbers?

Reply:
Thanks, yes.